lib/fb-jwt-client.js

```diff
@@ -165,10 +170,16 @@ class FBJWTClient {
    * Access token
    *
    **/
-  generateAccessToken (data) {
+  generateAccessToken (data, secret, algorithm) {
```

Comment:
What happens if `algorithm` is undefined?

Reply:
the library will probably throw an error. this should be a private method, i'm not sure why it's being directly tested